package.json

```diff
@@ -59,10 +59,15 @@
       "require": "./dist/svgo-node.cjs",
       "types": "./lib/svgo-node.d.ts"
     },
+    "./dist/svgo.browser.js": {
```

Comment:
If I'm not mistaken, that this will still fail if one requires `dist/svgo.browser`. We might need to tweak it.